addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -1407,13 +1386,13 @@ test("Channel is added to discuss after invitation", async () => {
             channel_member_ids: [Command.create({ partner_id: partnerId })],
         },
     ]);
-    const { env } = await start();
+    const env = await start();
     await openDiscuss();
     await contains(".o-mail-DiscussSidebarChannel", { text: "my channel" });
     await contains(".o-mail-DiscussSidebarChannel", { count: 0, text: "General" });
-    pyEnv.withUser(userId, () =>
+    withUser(userId, () =>
         env.services.orm.call("discuss.channel", "add_members", [[channelId]], {
-            partner_ids: [pyEnv.adminPartnerId],
+            partner_ids: [serverState.partnerId],
```

Comment:
This changed from admin to partner, isn't the withUser impacting what is in serverState?

Reply:
Probably, test passes regardless... I reverted to `serverState.partnerId` before `withUser`, as `adminPartnerId` doesn't exist anymore